pkg/kopia/command/parse_command_output.go

```diff
@@ -153,7 +153,7 @@ func sumSnapshotSizes(snapList []*snapshot.Manifest) (sum int64) {
 	return sum
 }
 
-func parseSnapshotManifestList(output string) ([]*snapshot.Manifest, error) {
+func ParseSnapshotManifestList(output string) ([]*snapshot.Manifest, error) {
```

Comment:
Are we calling this function from any other package? If not, then we should not export it.

Reply:
Thanks ! made the changes

Reply:
@r4rajat We may use it in other packages soon. We can keep it exported.

Reply:
Okay Will do that